docs/api/app.md

```diff
@@ -506,6 +506,41 @@ gets emitted.
 **Note:** Extra command line arguments might be added by Chromium,
 such as `--original-process-start-time`.
 
+### Event: 'notification-activation' _macOS_ _Windows_
+
+Returns:
+
+* `event` Event
+* `appUserModelId` string - contains App module ID
+* `invokedArgs` string - contains arguments(in case of any) for toast button or data property in case of non-persistent notification
+* `dataCount` integer - contains amount of token pairs present inputData string (not null, default value 0 - which means abcense of inputData)
+* `inputData` string - valid JSON-string which contains result of User input into toast Reply fields. Empty string in case of absence of User input (i.e for case dataCount == 0)
+
+Emitted after each completed user choise on notification.
```

Comment:
```suggestion
Emitted after each completed user choice on notification.
```

Reply:
Thank You for pointing this mistake. But I do not think that typo mistake into not merged code base is a big issue(unfortunately I'm not native English speaker)